### ISSUE TITLE
implementare la gestione delle radici generiche

### ISSUE THREAD

**Issue by knickita** [integr-abile/audiofunctions2.0#2]
ad oggi possiamo solo fare radici quadrate.
ad esempio non c'è modo d fare la radice cubica di 12..

**Comment by knickita** [comment]
function plot supporta le radici generiche.
ad esempio per esprimere la radice cubica di X si scrive `nthRoot(x,3)`
ora bisogna capire come gestire questa cosa da _**mathlive**_

**Comment by knickita** [comment]
[qui](https://cortexjs.io/mathlive/reference/commands/) è indicato che le radici generiche sono supportate da _**mathlive**_.
usando il comando `\sqrt[3]{x}` ad esempio si ottiene la radice cubica di x.
il problema è che appena si è cominciato a scrivere sqrt, viene subito creta la radice quadrata, senza dare il tempo di specificare nient'altro

**Comment by knickita** [comment]
se si fa copia e incolla del comando` \sqrt[3]{x}` direttamente nella funzione, si comporta tutto nel modo corretto..

bisogna capire perchè non è possibile scrivere il comando un carattere alla volta, ma si se copia tutto in blocco funziona

**Comment by Mattia541993** [comment]
Appena riesco vedo se capisco come fare. Questi test li stai facendo su un branch specifico?

**Comment by knickita** [comment]
no, li ho fatto su un pò di branch vari, ma il risultato è sempre lo stesso, anche perchè credo sia dovuto all'interpretazione di quello che scriviamo da parte di mathlive e non abbiamo ancora mai fatto una configurazione particolare di quel componente..

**Comment by Mattia541993** [comment]
Allora lasciamo così com'è. Scrivi questo problema in un task nella wish list del nostro Kanban e poi lo gestiremo. Quando sei pronto apri una merge request che le do un'occhiata ok?
